README.md

```diff
@@ -1,12 +1,12 @@
-# WRROC <> GA4GH API Conversion Tool
+# CrateGen
 [![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](./LICENSE)
 [![Python 3.11](https://img.shields.io/badge/python-3.11-blue.svg)](https://www.python.org/downloads/release/python-311/)
 
 ## Synopsis
 This project aims to create a library that translates between GA4GH Cloud API schemas and RO-Crate profiles to facilitate reproducibility in scientific research. The tool will enable the packaging and sharing of task/tool and workflow runs, allowing other researchers to reproduce them or use parts of them.
 
 ## Description
-The WRROC <> GA4GH API Conversion Tool is designed to compile RO-Crates from tool and workflow executions in GA4GH Cloud API-powered cloud infrastructures. This will allow easy re-execution of these computational analyses, thus improving the reporting and reproducibility of computational research.
+The CrateGen Tool is designed to compile RO-Crates from tool and workflow executions in GA4GH Cloud API-powered cloud infrastructures. This will allow easy re-execution of these computational analyses, thus improving the reporting and reproducibility of computational research.
```

Comment:
Note that it should also allow the opposite: creating WES/TES calls from existing WRROCs. But fine to keep it for now until that functionality is implemented (but should then also be clarified in the synopsis).

Reply:
I will design the tool to create complete valid RO-Crate files. If an RO-Crate already exists, the tool will add the packaged WRROC entity to the existing file.